include/openenclave/internal/load.h

```diff
@@ -6,6 +6,7 @@
 
 #include <openenclave/bits/defs.h>
 #include <openenclave/bits/result.h>
+#include <openenclave/bits/sgx/region.h>
```

Comment:
Is this file SGX-specific?  If so, it's in the wrong directory (should be moved down a level under sgx)